packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -5346,6 +5346,38 @@ void main() {
           throwsAssertionError,
         );
       });
+
+      testWidgets('InputDecorator throws when error and errorBuilder are provided', (WidgetTester tester) async {
```

Comment:
And when you add the assertion mentioned above, please also add another test like this that covers the new assertion.

Reply:
Shouldn't there be another test like this?: "InputDecorator throws when error and errorText are provided"

Reply:
+1 to this, it'd be nice to have full test coverage here.